tools/packaging/kata-deploy-cc/Dockerfile

```diff
@@ -3,8 +3,9 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Specify alternative base image, e.g. clefos for s390x
-ARG IMAGE
-FROM ${IMAGE:-registry.centos.org/centos}:7
+ARG IMG_NAME
+ARG IMG_TAG
+FROM ${IMG_NAME:-registry.centos.org/centos}:${IMG_TAG:-7}
```

Comment:
This is for a follow-up discussion and should not block this PR from being merged.

It is uses a quite old centos/clefos 7 for the favor of installing kubectl from a repository. I wonder whether we should go to a more recent base image version and install a more recent [`kubectl`](https://storage.googleapis.com/kubernetes-release/release/).

@BbolroC @fidencio Wdyt?

Reply:
@hbrueckner, the whole work is not exactly in favour of installing kubectl from a repo, but rather in favour of using systemd inside the container.  AFAIR we couldn't get to work with newer a newer OS.

If you manage to do so, and that is reliable for all the architectures, I'd be fine on having this in as a separate PR.
Makes sense?

Reply:
@fidencio I was not aware of those systemd issues and I have it added to my todo list to follow-up. If you have any references to prior systemd failures, please let me know. Thanks for clarification.

Reply:
@hbrueckner I would like to resolve the conversation if no further progress is made on your questions. We could continue the discussion with a separate PR

Reply:
@BbolroC  Please mark the conversation above as resolved. Changing and gettting insights into the systemd issues / newer base image is separate issue/PR. Thanks!